packages/i18n-calypso/README.md

```diff
@@ -124,7 +124,7 @@ const example2 = i18n.translate( 'I feel {{em}}very{{/em}} strongly about this.'
 // components can nest
 const example3 = i18n.translate( '{{a}}{{icon/}}click {{em}}here{{/em}}{{/a}} to see examples.', {
 	components: {
-		a: <a href="#" />,
+		a: <a href="http://example.com" />,
```

Comment:
Ideally, we're trying to use domains we own in documentation. You can use `https://yourjetpack.blog` or `https://wordpress.com/`

Reply:
example.com is actually a reserved second-level domain for this purpose. See https://tools.ietf.org/html/rfc2606

Reply:
Fair enough 👍